debian/changelog

```diff
@@ -0,0 +1,5 @@
+mu (0.1-1) unstable; urgency=low
```

Comment:
It's not immediately clear what is going wrong here. Nothing you're doing seems obviously wrong. What does the newly added entry to the changelog look like?